.github/workflows/main.yml

```diff
@@ -64,14 +64,35 @@ jobs:
         run: |
           ./CI/check-cmake.sh
 
+  config:
+    name: '01 - Configure Build Jobs'
+    runs-on: [ubuntu-22.04]
+    outputs:
+      create_artifacts: ${{ steps.config.outputs.create_artifacts }}
+      cache_date: ${{ steps.config.outputs.cache_date }}
+    steps:
+      - name: 'Configure Build Jobs'
+        id: config
+        run: |
+          if [[ "${{ github.event_name == 'pull_request' }}" == "true" ]]; then
+            if test -n "$(curl -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" -s "${{ github.event.pull_request.url }}" | jq -e '.labels[] | select(.name == "Seeking Testers")')"; then
```

Comment:
Can be replaced by using `${{ contains(github.event.pull_request.labels.*.name, 'Seeking Testers') }}`, which should yield `true` if the PR has a label of that name.

Reply:
Change done.